CONTRIBUTING.md

```diff
@@ -74,11 +74,14 @@ If you need more context on a particular issue, please ask.
 Remember, if you are unsure about anything, don't hesitate to comment on issues
 and ask for clarifications!
 
+**The _TODO_** comment<pr>
+
+Use the _TODO_ comment to capture technical debt.  It should not be used to flag areas that need to be fixed.  FixMe blocks need to be cleaned up before code is submitted.   
```

Comment:
I thought we had defined these the other way when we discussed this a few months ago (TODO should be cleaned up before code is submitted, FIXME to define technical debt).  But I may be remembering wrong.

Reply:
ooops.  Fixing that now